uint32  req_mode
---
uint32  current_mode
